### PR TITLE
PD-359: KVDB spins up an additional disk in cloud

### PR DESCRIPTION
**Pull request by andreipope**
PD-359: KVDB spins up an additional disk in cloud

### CODE REVIEW

File: content/concepts/internal-kvdb.md [hunk 1]
```diff
@@ -1,55 +1,59 @@
...
+Starting with version 2.0, you can install Portworx with a built-in internal KVDB. Portworx automatically deploys the internal KVDB cluster on a set of three nodes in your cluster and removes the requirement for an external KVDB such as etcd or Consul.
 
-## Setup
+## Install
 
-Portworx, when installed with appropriate arguments, will automatically deploy an internal kvdb cluster on a set of 3 nodes within the Portworx cluster. Based on your scheduler provide the appropriate input argument to setup Portworx with internal kvdb
+To install Portworx with a built-in internal KVDB, follow the instructions in the next sections, depending on your environment.
+
+{{<info>}}
+**NOTE:** Cloud deployments require an additional 150GB SSD disk for the internal KVDB, overriding the **"SKIP MetaDevice option"**. <!-- I don't really get it. Should we rather say something like "this is different from the cache drive"?  Or maybe just drop this.-->
```
Comment (andreipope) [inline content/concepts/internal-kvdb.md:16]: @sanjaynaikwadi @adityadani

Comment (andreipope) [reply content/concepts/internal-kvdb.md:16]: @sanjaynaikwadi @adityadani Just a kind reminder.

Comment (adityadani) [reply content/concepts/internal-kvdb.md:16]: We want to mention that when using internal kvdb on Cloud deployments, Portworx will create a new Kvdb Drive of size 150G if a separate kvdb device argument is not provided at install time. This is different than the cache device.

File: content/concepts/internal-kvdb.md [hunk 2]
```diff
@@ -1,55 +1,59 @@
...
 #### Other schedulers
 
-For all other schedulers, Portworx requires an external etcd or consul, to bootstrap its internal kvdb. Portworx will use this external etcd or consul, to discover its own internal kvdb nodes. While installing Portworx provide the `-b` argument to instruct Portworx to setup internal kvdb. Along with `-b`, provide the `-k` argument with a list of external etcd or consul endpoints.
+For all other schedulers, Portworx requires an external etcd or consul, to bootstrap its internal KVDB. Portworx will use this external etcd or consul, to discover its own internal KVDB nodes. While installing Portworx provide the `-b` argument to instruct Portworx to setup internal KVDB. Along with `-b`, provide the `-k` argument with a list of external etcd or consul endpoints.
 
 
-## Storage for internal kvdb
+## Storage for internal KVDB
 
-Internal kvdb metadata needs to be stored on a persistent disk. There are two ways in which internal kvdb can store this data
```
Comment (adityadani) [inline content/concepts/internal-kvdb.md:30]: The above Note for kvdb drive in cloud deployments should come under this section.

**Pull request merged**